### DISCUSSION TITLE
Convert RVC pth model to safetensors | RVCのpthモデルをSafetensorsに変換

### DISCUSSION THREAD

**Discussion by ashdarain** [litagin02/Style-Bert-VITS2#34] [Q&A]
Is there a way to use RVC models in Style-Bert-VITS2. Like converting it to Safetensors format?

Style-Bert-VITS2でRVCモデルを使用する方法はありますか？Safetensorsに変換する方法はありますか

**Comment by litagin02** [comment]
No, RVC is audio-to-audio, this is text-to-speech, the model architectures are completely different, so you cannot.
